### PR TITLE
Removed version filtering proposal and added discussion

### PR DESCRIPTION
**Pull request by diekhans**
Removed version filtering proposal and added discussion of issues with push notifications granular testing

### CODE REVIEW

**Comment by brianraymor** [comment]
@diekhans - is this a RFC in progress (not ready for community review)?

**Comment by diekhans** [comment]
Yes, It is in progress. Not ready to review 

-----Original Message-----

File: rfcs/text/0000-metadata-integration-test.md [hunk 1]
```diff
@@ -20,19 +20,20 @@ This document details a integration test design for metadata schema changes in t
...
+1. Increase availability of DCP systems downstream of DSS.
 
 ## Detailed Design
 
 In this design we will use the following terms:
  
- * "schema integration test system" - an automated CI/CD pipeline used to check that downstream systems can handle bundles with a new schema
- * "new schema label" - anything that indicates that the schema has changes that have not yet been tested against downstream systems: potentially a JSON field, git tag, or git branch
+ * "schema integration test system" - an automated CI/CD pipeline used to check that downstream systems can handle bundles with a new schema.
+ * "new schema label" - anything that indicates that the schema has changes that have not yet been tested against downstream systems: potentially a JSON field, git tag, git branch, or comparison to tested version list.
```
Comment (mweiden) [inline rfcs/text/0000-metadata-integration-test.md]: nit: I don't usually put periods at the end of sentences if they are not full sentences.

File: rfcs/text/0000-metadata-integration-test.md [hunk 2]
```diff
@@ -48,45 +49,30 @@ Design
...
-* Tests exactly what bundles downstream systems will see; does not require complicated fake metadata generation code in tests
-* Data in storage is immediately available after upload via the `GET /v1/bundles` endpoint
-* Downstream systems are protected from new schema changes until it can be confirmed that they are able to handle the changes
-* Does not require replaying subscription notifications
+* Bundles only have to be uploaded once.
+* Bundles can be uploaded directly to the production environment.
+* Tests exactly what bundles downstream systems will see; does not require complicated fake metadata generation code in tests.
+* Data in storage is immediately available after upload via the `GET /v1/bundles` endpoint.
+* Downstream systems are protected from new schema changes until it can be confirmed that they are able to handle the changes.
+* Does not require replaying subscription notifications.
```
Comment (mweiden) [inline rfcs/text/0000-metadata-integration-test.md]: Here too

File: rfcs/text/0000-metadata-integration-test.md [hunk 3]
```diff
@@ -48,45 +49,30 @@ Design
...
+* In the case that the integration test does not pass, developers and wranglers will have to exercise care in propagating fixes to production before releasing the new bundles downstream of the production DSS.
 
-Testing is done independently on each component. Once the components test pass with the updated metadata, it's version list is automatically updated and queued submissions are processed.
+## Discussion:
 
-This approach is a fast-path that complements the above designed for simple, incompatible changes.
+It would be highly desirable to make metadata available to components based on the individual component tests passing, as opposed to all of the test.  In particular, viability in the data browser before secondary analysis application can process the data would address the needs to make the primary data available as quickly as possible.
 
+Using the major version number of the last passing test for a component could work as a filter.  However, the push notification model of the DCP makes implementing this a complex undertaking.  It is suggested that supporting a more granular data visibility model be considered based on the experiences with the above implementation.
```
Comment (mweiden) [inline rfcs/text/0000-metadata-integration-test.md]: `| sed s/DCP/DSS/` to be more precise

**Review by mweiden — APPROVED** [review]
Just a couple if nits here.

File: rfcs/text/0000-metadata-integration-test.md (hunk 3 above)
Comment (diekhans) [reply rfcs/text/0000-metadata-integration-test.md]: thanks!

**Pull request merged**